source/fundamentals/crud/read-operations/text.txt

```diff
@@ -0,0 +1,274 @@
+===========
+Search Text
+===========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you will learn how to run a **text search** in MongoDB.
+
+You can use a text search to search documents for a term or a phrase. A
+term is a sequence of characters that excludes whitespace characters. A
+phrase is a sequence of terms that with any number of whitespace
+characters. You can use this for searching through unstructured text,
+which is text without a predefined format such as resumes and
+course catalogs. 
+
+In MongoDB, you can run a text search on a string field. A text
+search retrieves documents that contain the term or phrase in the
+specified field. 
```

Comment:
I would combine this with the previous paragraph because the text search you're describing there is not a general one, but rather specific to MongoDB.
E.g.
"In MongoDB, a text search retrieves documents that contain a matching term or phrase in one or more string fields."